docs/source/demos/text_input.ipynb

```diff
@@ -0,0 +1,343 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate on hold out\n",
+    "\n",
+    "Finally, we retrain the best pipeline on all of the training data and evaluate on the holdout"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can score the pipeline on the hold out data using the core objectives for binary classification problems"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline.score(X_holdout, y_holdout, objectives=evalml.objectives.get_core_objectives('binary'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, this model performs relatively well on this dataset, even on unseen data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Why encode text this way?\n",
+    "\n",
+    "To demonstrate the importance of text-specific modeling, let's train a model with the same dataset, without letting `AutoMLSearch` detect the text column. We can change this by explicitly setting the data type of the `'Message'` column in Woodwork."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork import DataTable\n",
+    "\n",
+    "X_train_datatable = DataTable(X_train, logical_types={'Message': 'Categorical'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_no_text = AutoMLSearch(problem_type='binary',\n",
+    "                      max_iterations=5,\n",
+    "                      optimize_thresholds=True)\n",
+    "\n",
+    "automl_no_text.search(X_train_datatable, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like before, we can look at the rankings and pick the best pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_no_text.rankings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline_no_text = automl_no_text.best_pipeline\n",
+    "best_pipeline_no_text.graph()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, changing the data type of the text column removed the `Text Featurization Component` from the pipeline"
```

Comment:
Suggestion:
* Split the code above into two cells: first `best_pipeline_no_text = automl_no_text.best_pipeline`, then `best_pipeline_no_text.graph()`
* Move this text below the first cell (best pipeline) but above the call to `graph()`

That way, the call to graph and the call to describe are back-to-back, and occur after your text description.